### PR TITLE
Update italian.xml

### PR DESCRIPTION
**Pull request by DVDAndroid**
- Added _DoSaveOrNot_ keyboard accelerators
- Added new strings
- Fixed ugly code format

Close this PR if you don't want to review it for exceeding code formatting

### CODE REVIEW

File: PowerEditor/installer/nativeLang/italian.xml [hunk 1]
```diff
@@ -18,9 +18,9 @@
                     <Item menuId="language" name="&amp;Linguaggio"/>
                     <Item menuId="settings" name="C&amp;onfigurazione"/>
                     <Item menuId="tools" name="&amp;Strumenti"/>
-                    <Item menuId="macro" name="Macro"/>
-                    <Item menuId="run" name="Esegui"/>
-                    <Item idName="Plugins" name="Plugin"/>
+                    <Item menuId="macro" name="M&amp;acro"/>
+                    <Item menuId="run" name="Ese&amp;gui"/>
+                    <Item idName="Plugins" name="P&amp;lugin"/>
```
Comment (xylographe) [inline PowerEditor/installer/nativeLang/italian.xml]: Apparently, the letter `l` is used twice as a menu keyboard accelerator — in `P&amp;lugin` and `&amp;Linguaggio`.
Perhaps, `&amp;Plugin`?

**Comment by DVDAndroid** [comment]
Fixed

**Comment by xylographe** [comment]
Oops, if I had not forgotten to check, I would have noticed the two missing items that have been added to the `<MISC ...>` section in v7.8.1 and v7.8.3  sooner,
```xml
<Item id="6348" name="Don't fill find field in Find dialog with selected word"/>
<Item id="6314" name="Use monospaced font in Find dialog (Need to restart Notepad++)"/>
```

**Comment by donho** [comment]
Make your modification compact - don't reformat source code in your request. It makes code review more difficult.
https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/CONTRIBUTING.md#guidelines-for-pull-requests

**Pull request closed without merging**